src/rgw/rgw_rest_log.cc

```diff
@@ -935,6 +984,8 @@ RGWOp *RGWHandler_Log::op_get() {
   } else if (type.compare("bucket-index") == 0) {
     if (s->info.args.exists("info")) {
       return new RGWOp_BILog_Info;
+    } else if (s->info.args.exists("status")) {
```

Comment:
maybe `sync-status`?

Reply:
i had used `status` for datalog and mdlog trim, so i'd rather keep them consistent